api/api.proto

```diff
@@ -395,6 +407,10 @@ message ProxySpec {
     S3ProxySpec s3_spec = 4;
     // PXDProxySpec is the spec for proxying a Portworx volume
     PXDProxySpec pxd_spec = 5;
+    // PureFileSpec is the spec for proxying a volume on pure_file backends
+    PureBlockSpec pure_block_spec = 6;
```

Comment:
I thought we ended up saying that passthrough wouldn't be implemented using "proxy volumes"? Is this:
a) a naming difference and "proxy" in `libopenstorage` means something different than in internal code, or
b) the way we're implementing it uses proxy functionality in `libopenstorage` and the difference is purely in internal code?

Reply:
it's just store pure backend info into the ProxySpec, and make everything clean and manageable instead of making new fields in VolumeSpec which is already pretty large now , just for internal usage, the sc parameter key is set to `backend` instead of the original proxy volume keys to reduce confusion, and on porx side, we don't go through the proxy volume filter.